services/src/pro/contexts/postgres.rs

```diff
@@ -461,7 +475,7 @@ where
                             email character varying (256),
                             real_name character varying (256),
                             active boolean NOT NULL
-                        );
+                        );                     
```

Comment:
```suggestion
                        );
```

Sure, let's just add some spaces for the stats :stuck_out_tongue:

Reply:
23e1d9ebc662381de5ae75709bbb725e618fa97b